brainstormforce/ultimate-addons-for-gutenberg
# UAG-446 - Fix: Tabs Block - Tabs button are not clickable.

### Description

- https://ua-gutenberg.atlassian.net/browse/UAG-446

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Tabs on the page. Select a different style.
- Click on the tabs button to check is it clickable in the editor as well as on the frontend.
- Layout does not break and everything works properly in both ends.

### Checklist:
